docs/developer/install_master.md

```diff
@@ -18,7 +18,7 @@ helm repo add keptn-dev https://charts-dev.keptn.sh
 7. For current master use `<current_version>-dev` images, for images built in a PR add `-PR-<PR_#>` to the tag,
 8. To install version 0.13.0-dev-PR-1234 
 ``` 
-helm upgrade --install keptn keptn-dev/keptn -n keptn --create-namespace --set=continuous-delivery.enabled=true --wait --version 0.13.0-dev-PR-1234
+helm upgrade --install keptn keptn-dev/keptn -n keptn --create-namespace --set=continuousDelivery.enabled=true --wait --version 0.13.0-dev-PR-1234
```

Comment:
let`s use a `0.16.1-dev-PR` build as an example, as the `continuousDelivery` values object was not available in previous versions

Reply:
fixed